src/layout/msWorkspace/msWindow.ts

```diff
@@ -373,8 +371,8 @@ export class MsWindow extends Clutter.Actor {
         );
         this.trackAppChanges();
         state.matchingInfo.appId = this.app.id;
-        this.placeholder = this.buildPlaceHolder();
-        this.msContent.placeholder = this.placeholder;
+        this.placeholder.setIcon(this.app.create_icon_texture(248));
```

Comment:
You are using this constant in two places. Extract to global named constant.

Reply:
done I hope the name isn't so bad this time !